src/import/util.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2024 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use crate::runtime::model::{Easing, Tweenable};
+use serde_json::Number;
+
+pub trait NumberExt {
```

Comment:
Should this be `pub(crate)`?

Reply:
Not needed, because `util` is not public.

Reply:
You're right, we should add the unreachable_pub lint